parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -235,6 +235,30 @@ if [ "${SHOULD_CONFIG_SWAP_FILE}" == "true" ]; then
     logs_to_events "AKS.CSE.configureSwapFile" configureSwapFile
 fi
 
+mkdir -p "/etc/systemd/system/kubelet.service.d"
+
+if [ "${NEEDS_CGROUPV2}" == "true" ]; then
+    tee "/etc/systemd/system/kubelet.service.d/10-cgroupv2.conf" > /dev/null <<EOF
+[Service]
+Environment="KUBELET_CGROUP_FLAGS=--cgroup-driver=systemd"
+EOF
+fi
+
+if [ "${NEEDS_CONTAINERD}" == "true" ]; then
+    tee "" > /dev/null <<'EOF'
```

Comment:
might be missing something, but should this really be empty?

Reply:
F I set auto merge too fast